.agents/skills/swarm-performance/SKILL.md

```
@@ -0,0 +1,66 @@
+---
+name: swarm-performance
+description: "SIMD optimization, connection pooling, batch APIs, and caching. Use when improving throughput or reducing latency."
+---
+
+# Swarm: Performance
+
+## Workflow
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)